Procfile.CF

```diff
@@ -0,0 +1,2 @@
+web: bundle exec rake db:migrate && bundle exec rails server -p $PORT
```

Comment:
It is possible, but this version is a bit more comfortable and stable. The alternative would be to deploy a second background app, which is doing the migration and terminates afterwords. The downside of this solution is, both apps are started in parallel, so if the migration is not done in time, the server startup will fail in the first shot. 

The way it is implemented now will add just a couple of seconds to the startup process if the database already got the right version it will don nothing, otherwise it will run the update.